core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -5185,11 +5183,8 @@ components:
         - receipt_tree_hash
       properties:
         state_version:
-          type: integer
-          format: int64
-          minimum: 0
-          maximum: 100000000000000
-          description: An integer between `0` and `10^13`, representing the state version. The state version increments with each transaction, starting at `0` pre-genesis.
+          $ref: "#/components/schemas/StateVersion"
+          description: The state version, which increments with each transaction, starting at `0` pre-genesis.
```

Comment:
So, _technically_ this is invalid in the Open API schema language - ie you can't override any properties (including an upstream description) here. (Can you tell I'm not a fan of the Open API schema design and ecosystem? :') )

If you notice - the description has now vanished from all the models.

_But_ - importantly - I believe ReDocly _does_ handle this correctly, so this _should_ be fine. Although I'm not 100% sure. It might be worth us double-checking when we deploy the schema file.

Reply:
whooops, indeed o_O

Let's hope it renders the public-facing file fine - if not, ofc I will bring back the boilerplate 👍